src/Microsoft.Framework.Logging/Logger.cs

```diff
@@ -31,7 +31,14 @@ public void Log(LogLevel logLevel, int eventId, object state, Exception exceptio
             {
                 foreach (var logger in _loggers)
                 {
-                    logger.Log(logLevel, eventId, state, exception, formatter);
+                    try
+                    {
+                        logger.Log(logLevel, eventId, state, exception, formatter);
+                    }
+                    catch
+                    {
+                        // ignore exceptions thrown by a logger and continue calling other loggers
```

Comment:
Need to discuss with @lodejard and @DamianEdwards because I'm not quite sure what the best behavior is. It might be best to let the whole system go down in flames. My concern is that if a logger is being used for business purposes or logging important security information (e.g. a failed audit) and that logger throws an exception, should the app continue to run as if nothing happened? Or should the one bad logger be ignored? What if that was the only logger and now the app goes silent?